controllers/stardpginstance_reconciliator.go

```diff
@@ -0,0 +1,77 @@
+package controllers
```

Comment:
typo in the file name: `controllers/stardpginstance_reconciliator.go `

Besides, most of the functions defined here can also be used for the User and Role objects. I think it's worth making it reusable for other CRDs as well

Reply:
They will be reused of course when the the right time comes